react/src/pages/AgentSummaryPage.tsx

```diff
@@ -46,9 +46,7 @@ const ResourcesPage: React.FC<ResourcesPageProps> = (props) => {
             />
           }
         >
-          <AgentSummaryList
-            containerStyle={{ marginLeft: -1, marginRight: -1 }}
-          />
+          <AgentSummaryList />
```

Comment:
Is there any reason to modify this file?

Reply:
No.. I'll take it back.